syft/internal/cache/test_cache.go

```diff
@@ -0,0 +1,11 @@
+package cache
+
+import "testing"
+
+func NewTestCache(t *testing.T) {
```

Comment:
This should probably be moved into its own package, something like `cachetest` (so something like `cachetest.New()` or `cachetest.NewCache()`) otherwise the syft binary will start depending on the stdlib `testing` package. But if separated, then we'll only import `cachetest` within `_test.go` files, thus there would not be a direct dependency on `testing`.

This is not a critical comment since this is not a public package. the only hesitation is that we have public packages that depend on this package which directly uses `testing`.

Reply:
I've just removed this, it was no longer needed for anything